crates/cheatnet/tests/cheatcodes/prank.rs

```diff
@@ -43,7 +43,6 @@ fn prank_with_other_syscall() {
 }
 
 #[test]
-#[ignore = "TODO(#254)"]
```

Comment:
There are more constructor tests

Reply:
Added

Reply:
BTW the mock call one was broken, so i fixed it